### ISSUE TITLE
HDF5 trainvaltest split for PACS

### ISSUE THREAD

**Issue by YiDongOuYang** [microsoft/robustdg#13]
Could you release the hdf5 of PACS? I would like to make several improvements to matchDG. Thanks in advance!

**Comment by divyat09** [comment]
You can download the hdf5 of PACS from here: https://drive.google.com/drive/folders/0B6x7gtvErXgfUU1WcGY5SzdwZVk

**Comment by YiDongOuYang** [comment]
Dear Divyat,

I found I cannot reproduce the results on PACS. I used the same command mentioned in #2. The first image below is ERM algorithm (95.84% mentioned in the paper for ResNet18 when the unseen domain is Photo, while I only got 86.55%). The second image is matchdg_ctr.

Thanks in advance!

<img width="1093" alt="PACS_ERM" src="https://user-images.githubusercontent.com/58366413/102898039-e5299e00-44a3-11eb-902b-8de611c536a5.png">
<img width="1108" alt="PACS_matchdg_ctr" src="https://user-images.githubusercontent.com/58366413/102898049-e6f36180-44a3-11eb-8d21-cf74d98b530a.png">

**Comment by divyat09** [comment]
Its because of hyperparameter tuning, the learning rate for the domain Photo should be 0.0005, which is different from the default in the library. Please refer to table 6 & 7 in this paper for details regarding hyperparams: https://arxiv.org/pdf/2006.07500.pdf

Thanks, I will also update the readme command. 

Note: The current code in the master version does not show accuracy using different validation criteria. If you want to obtain test accuracy under different validation mechanism, refer to the code in this [branch ](https://github.com/microsoft/robustdg/tree/mia_adv_attacks), which will be soon merged with the master also.
